content/20.pharmaceuticals.md

```diff
@@ -0,0 +1,392 @@
+It wasn't until February 11, 2021 that a preprint describing the first randomized control trial of TCZ was released as part of the RECOVERY trial [@doi:10.1101/2021.02.11.21249258].
+The first randomized control trial of TCZ to report results was the RECOVERY trial [@doi:10.1101/2021.02.11.21249258], where, 2,022 hospitalized patients in the U.K. were randomized to receive TCZ and 2,094 were randomized to SOC, with 79% of patients in each group available for analysis at the time that the initial report was released.
+TCZ was found to reduce 28-day mortality, from 33% in patients receiving SOC alone to 29% in those receiving TCZ.
+Combining the RECOVERY trial data with data from seven smaller randomized control trials indicates that TCZ is associated with a 13% reduction in 28-day mortality (rate ratio 0.87, 95% CI 0.79-0.96, _p_ = 0·005) [@doi:10.1101/2021.02.11.21249258].
+While this initial report did not include the full results expected from the RECOVERY trial, this large-scale, randomized controlled trial therefore offers strong evidence that TCZ may offer benefits for COVID-19 patients, even at this initial stage of analysis.
+However, the fact that TCZ suppresses the immune response means that it does carry risks for patients, especially a potential risk of secondary infection (see Appendix XX).
+
+TCZ is just one example of a candidate drug targeting the host immune response, and specifically excessive inflammation.
+For example, interferons (IFNs) have also been investigated; these are a family of cytokines critical to activating the innate immune response against viral infections.
+Synairgen has been investigating a candidate drug, SNG001, which is an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095] that they reported reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@synairgen-SNG001-July-2020; @doi:10/ghjzm4].
+However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (see Appendix XX).
+Other drugs influencing inflammation are also being explored (see Appendix XX).<!-- To Do: Figure out what to cite from drugs in the potential alternate section-->
+It is also important that studies focused on inflammation as a possible therapeutic target consider the potential differences in baseline inflammation among patients from different backgrounds and with differing life experiences (see [@individual-inequality]).
+
+#### Biologics and the Adaptive Immune Response
```

Comment:
I think that we used to refer to a review on convalescent plasma that we may want to bring back. My understanding is that, in practical terms, it doesn't really work for this disease (though some will argue that in impractical settings it may be helpful). I feel like we should at least bring back a brief mention and point to one of the other reviews on the topic.

Reply:
Good point! @nilswellhausen had added a sentence in #876 that I think might be what you're referring to:

> While polyclonal antibodies from convalescent plasma can be used as a treatment for COVID-19, this section focuses on current efforts in developing monoclonal neutralizing antibodies (nAbs) against SARS-CoV-2 (excellent reviews regarding convalescent plasma therapy can be found here [@doi:10/ggqr7s; @doi:10.1001/jama.2020.4940]). 

I added a closely derived sentence to the summary for this section.